stacks/nodejs-mongodb/devfile.yaml

```diff
@@ -0,0 +1,77 @@
+schemaVersion: 2.2.2
+metadata:
+  name: nodejs-mongodb
+  displayName: Node.js MongoDB
+  description: NodeJS web application which communicates with MongoDB
+  icon: https://github.com/che-samples/nodejs-mongodb-sample/raw/main/nodejs-mongodb.png
+  tags:
+    - Node.js
+    - MongoDB
+    - ubi9
+  projectType: universal
+  language: Polyglot
+  version: 1.0.0
+projects:
+  - name: nodejs-mongodb-sample
+    git:
+      remotes:
+        origin: 'https://github.com/che-samples/nodejs-mongodb-sample'
+      checkoutFrom:
+        revision: main
+components:
+  - name: tools
+    container:
+      # quay.io/devfile/universal-developer-image:ubi9-dd1f49f
+      image: quay.io/devfile/universal-developer-image@sha256:c7ef40c8e6997d8961572a0e9088aec6735b4df0dedae2361085a44260b8c12f
+      env:
+      # The values below are used to set up the environment for running the application
+        - name: SECRET
+          value: 220fd770-c028-480d-8f95-f84353c7d55a 
```

Comment:
Here, if possible, could we use maybe a "dummy" string so is obvious to the user that they need to change it. Also if is meant to be used for production might be better to think about an outerloop approach?

cc @michael-valdron what do you think?

Reply:
set dummy value